src/test/java/org/apache/mesos/dcos/CapabilitiesTest.java

```diff
@@ -12,7 +12,6 @@
  */
 public class CapabilitiesTest {
```

Comment:
@nickbp:  Thanks for all the tests.  Can you add one more that exercises the throwing of the NumberFormatException please?  So something that has version returned that looks like `1` for example and maybe something with `0.` as another example.